packages/admin/dashboard/src/i18n/translations/pl.json

```diff
@@ -0,0 +1,2834 @@
+{
+    "$schema": "./$schemat.json",
```

Comment:
```suggestion
    "$schema": "./$schema.json",
```

Reply:
Once you have pointed the file to the schema, your editor should be able to tell you all the keys that are unexpected or missing.